Exec November 2: Draft (#2012)
makerdao/community: governance/votes/Executive vote - November 2, 2022.md

* **167,666 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at [0x3D274fbAc29C92D2F624483495C0113B44dBE7d2](https://etherscan.io/address/0x3D274fbAc29C92D2F624483495C0113B44dBE7d2) as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
* **43,332 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at [0xc657aC882Fb2D6CcF521801da39e910F8519508d](https://etherscan.io/address/0xc657aC882Fb2D6CcF521801da39e910F8519508d) as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
* **26.04 MKR** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at [0xc657aC882Fb2D6CcF521801da39e910F8519508d](https://etherscan.io/address/0xc657aC882Fb2D6CcF521801da39e910F8519508d) as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).

> * **143.46 MKR** will be transferred to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) auditor wallet at [0xD1505ee500791490DE8642353BA6A5b92e3550F7](https://etherscan.io/address/0xD1505ee500791490DE8642353BA6A5b92e3550F7) as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).

Reviewer: this is my bad, I put the wrong address in the sheet - should be 0x96d7b01Cc25B141520C717fa369844d34FF116ec

```suggestion
* **143.46 MKR** will be transferred to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) auditor wallet at [0x96d7b01Cc25B141520C717fa369844d34FF116ec](https://etherscan.io/address/0x96d7b01Cc25B141520C717fa369844d34FF116ec) as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).
```

---

> title: Template - [Executive Vote] Maker Teleport Deployment on Starknet, Offboarded Core Units Budgets and Streams, Oracle Whitelist Additions, and BlockTower SPF Funding - November 02, 2022

Reviewer: ```suggestion
title: Template - [Executive Vote] Maker Teleport Technical Deployment on Starknet, Offboarded Core Units Budget Stream Housekeeping, Oracle Whitelist Addition, and SPF Funding - November 02, 2022
```

> summary: Maker Teleport Technical Deployment on Starknet, Golden Parachute MKR and DAI budget transfers for EVENTS-001, SH-001, and RWF-001, yank MKR and DAI budget streams for EVENTS-001, SH-001, and RWF-001, whitelist Oasis.app on rETH/USD oracle, and BlockTower Legal Work and Risk Work SPF Funding. 

Reviewer: ```suggestion
summary: Maker Teleport Technical Deployment on Starknet, Golden Parachute MKR and DAI budget transfers for EVENTS-001, SH-001, and RWF-001, yank MKR and DAI budget streams for EVENTS-001, SH-001, and RWF-001, whitelist Oasis.app on rETH/USD oracle, and BlockTower Legal and Risk Work SPF Funding. 
```

date: 2022-11-02T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] Maker Teleport Deployment on Starknet, Offboarded Core Units Budgets and Streams, Oracle Whitelist Additions, and BlockTower SPF Funding - November 02, 2022

Reviewer: ```suggestion
# [Executive Proposal] Maker Teleport Technical Deployment on Starknet, Offboarded Core Units Budget Stream Housekeeping, Oracle Whitelist Addition, and SPF Funding - November 02, 2022
```

> The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

Reviewer: I tend to credit technical teams where relevant

```suggestion
The Governance Facilitator(s), the Starknet Engineering Core Unit, and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.

---

## Executive Summary

If this executive proposal passes, the following **changes** will occur within the Maker Protocol:

> - The Maker Teleport Global [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased by **1 million DAI** to **3 million DAI**.

Reviewer: This is just a summary it's ok to be less detailed here

```suggestion
- The Maker Teleport Global [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased, as detailed below.
```

Author: I've committed your change but just checking that the global debt ceiling for Teleport is now 3 million and not 2,100,000? Just thought that might be worth clarifying

> - A total of **210,998 DAI** will be transferred to multiple offboarded Core Units as part of their respective Golden Parachutes.
> - A total of **169.5 MKR** will be transferred to multiple offboarded Core Units as part of their respective Golden Parachutes.
> - DAI streams for various now offboarded will be yanked.
> - The MKR stream for the now offboarded [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be yanked.
> - [Oasis.app](https://oasis.app) will be whitelisted on the rETH/USD oracle.
> - **258,000 DAI** will be transferred to the BlockTower Legal and Risk Work SPF multi-sig as a result of the successful [Special Purpose Fund (SPF) application](https://vote.makerdao.com/polling/QmcV2pMZ#vote-breakdown).

Reviewer: 1. try and keep things human readable
2. this is a summary, we will add more details below

```suggestion
- A total of **210,998 DAI**, itemized below, will be transferred to two offboarded Core Units as part of their respective Golden Parachutes.
- A total of **169.5 MKR**, itemized below, will be transferred to two offboarded Core Units as part of their respective Golden Parachutes.
- DAI streams to three now offboarded Core Units, itemized below, will be cancelled.
- The MKR stream to the now offboarded [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be cancelled, as detailed below.
- [Oasis.app](https://oasis.app) will be whitelisted on the rETH/USD oracle, as detailed below.
- **258,000 DAI** will be transferred to the BlockTower Legal and Risk Work SPF multi-sig as a result of the successful [Special Purpose Fund (SPF) application](https://vote.makerdao.com/polling/QmcV2pMZ#vote-breakdown), as detailed below.
```

If this executive proposal passes, the following **additions** will be made to the Maker Protocol:

> - Maker Teleport will be deployed with **1 Basis Point** fee, a **1000 DAI** Deposit Limit, and **1 million DAI** Debt Ceiling on [Starknet](https://starkware.co/starknet/).

Reviewer: Summary

```suggestion
- The technical deployment of Maker Teleport on [Starknet](https://starkware.co/starknet/) will take place, as detailed below.
```

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

### Maker Teleport Technical Deployment on Starknet

> As per the outcome of successful Governance Polls [here](https://vote.makerdao.com/polling/QmbWkTvW) and [here](https://vote.makerdao.com/polling/QmZxRgvG), [Maker Teleport (formerly known as Maker Wormhole)](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) will be deployed on [Starknet](https://starkware.co/starknet/), if this executive passes. The Maker Teleport parameters will be set as follows:

Reviewer: add new line
```suggestion
As per the outcome of successful Governance Polls [here](https://vote.makerdao.com/polling/QmbWkTvW) and [here](https://vote.makerdao.com/polling/QmZxRgvG), [Maker Teleport (formerly known as Maker Wormhole)](https://forum.makerdao.com/t/introducing-maker-wormhole/11550) will be deployed on [Starknet](https://starkware.co/starknet/), if this executive passes.

The Maker Teleport parameters will be set as follows:
```

* The Starknet Teleport Fee parameter will be set to **1 Basis Point**.

> * The Deposit Limit will be set to **1000 DAI**.

Reviewer: Remove - looks like Teleport doesn't have a deposit limit - see https://discord.com/channels/893112320329396265/897483518316265553/1035271830354804846

```suggestion
```

> * The [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to **1 million DAI**.

Reviewer: see - https://discord.com/channels/893112320329396265/897483518316265553/1035271830354804846

```suggestion
* The [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to **100,000 DAI**.

**Note:** to ensure security of the deployment this is below the 1 million DAI limit approved by Maker Governance, this limit can be raised later, if required.
```

> Please review [MIP69](https://mips.makerdao.com/mips/details/MIP69) for details on Layer-2-to-Layer-1 Fast Withdrawals and the [discussion thread](https://forum.makerdao.com/t/request-for-poll-starknet-bridge-deposit-limit-and-starknet-teleport-fees/17187) on  Starknet parameters to inform your position on this issue before voting.

Reviewer: Remove double space

```suggestion
Please review [MIP69](https://mips.makerdao.com/mips/details/MIP69) for details on Layer-2-to-Layer-1 Fast Withdrawals and the [discussion thread](https://forum.makerdao.com/t/request-for-poll-starknet-bridge-deposit-limit-and-starknet-teleport-fees/17187) on Starknet parameters to inform your position on this issue before voting.
```

> ### Offboarded Core Units - Yanking DAI and MKR Streams

Reviewer: Add line space, use human readable name where possible

```suggestion
### Offboarded Core Units - Cancelling DAI and MKR Streams

```

> As per multiple successful offboarding proposals (linked below), several DAI and MKR streams will be yanked if this executive proposal passes. Note that when a token stream is yanked, it will prevent new tokens from accruing to the recipient, but will still allow the recipient access to tokens that have already accrued.

Reviewer: In this section we can be specific 😄 

```suggestion
As per multiple successful offboarding proposals (linked below), three DAI and one MKR streams will be cancelled (`yank`ed) if this executive proposal passes. 

When a token stream is `yank`ed, it will prevent new tokens from accruing to the recipient, but will still allow the recipient access to tokens that have already accrued.
```

> * **DAI Stream #4** going to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) will be yanked as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
> * **DAI Stream #5** going to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be yanked as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
> * **MKR Stream #23** going to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be yanked as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
> * **DAI Stream #35** going to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) will be yanked as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).

Reviewer: Mostly fine, I would do all DAI streams together and then all MKR streams but this is a style thing really

```suggestion
* **DAI Stream #4** going to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) will be cancelled as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
* **DAI Stream #5** going to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be cancelled as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
* **DAI Stream #35** going to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) will be cancelled as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).
* **MKR Stream #23** going to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) will be cancelled as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
```

Please review [MIP39](https://mips.makerdao.com/mips/details/MIP39) for details on offboarding Core Units and the forum threads discussing the [offboarding of the Events Core Unit (EVENTS-001)](https://forum.makerdao.com/t/mip39c3-sp4-offboarding-the-events-core-unit-events-001/17755), the [offboarding of the Strategic Happiness Core Unit (SH-001)](https://forum.makerdao.com/t/mip39c3-sp3-offboarding-the-strategic-happiness-core-unit-sh-001/17754), and the [offboarding of the Real-World Finance Core Unit (RWF-001)](https://forum.makerdao.com/t/mip39c3-sp5-offboarding-the-real-world-finance-core-unit-rwf-001/17753) to inform your position on this issue before voting.

> 
> 
> 

Reviewer: remove extra empty lines

```suggestion
```

### Offboarded Core Units - DAI and MKR Golden Parachute Budget Transfers

As per multiple successful offboarding proposals (linked below), a total of **210,998 DAI** and **169.5 MKR** will be transferred to multiple offboarded Core Units as part of their respective Golden Parachutes if this executive proposal passes.
* **167,666 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at 0x3D274fbAc29C92D2F624483495C0113B44dBE7d2 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
* **43,332 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
* **26.04 MKR** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).

> * **143.46 MKR** will be transferred to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) at 0xD1505ee500791490DE8642353BA6A5b92e3550F7 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).

Reviewer: add etherscan links to wallet addresses

> * **167,666 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at 0x3D274fbAc29C92D2F624483495C0113B44dBE7d2 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
> * **43,332 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
> * **26.04 MKR** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
> * **143.46 MKR** will be transferred to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) at 0xD1505ee500791490DE8642353BA6A5b92e3550F7 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).

Reviewer: specify this is the auditor wallet

```suggestion
* **167,666 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at 0x3D274fbAc29C92D2F624483495C0113B44dBE7d2 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
* **43,332 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
* **26.04 MKR** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
* **143.46 MKR** will be transferred to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) auditor wallet at 0xD1505ee500791490DE8642353BA6A5b92e3550F7 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).
```

> Please review [MIP39](https://mips.makerdao.com/mips/details/MIP39) for details on offboarding Core Units and the forum threads discussing the [offboarding of the Events Core Unit](https://forum.makerdao.com/t/mip39c3-sp4-offboarding-the-events-core-unit-events-001/17755), the [offboarding of the Strategic Happiness Core Unit](https://forum.makerdao.com/t/mip39c3-sp3-offboarding-the-strategic-happiness-core-unit-sh-001/17754), and the [offboarding of the Real-World Finance Core Unit](https://forum.makerdao.com/t/mip39c3-sp5-offboarding-the-real-world-finance-core-unit-rwf-001/17753) to inform your position on this issue before voting.

Reviewer: add link to my summary thread - https://forum.makerdao.com/t/executive-vote-cu-offboarding-next-steps/18522

### Oasis.app Whitelist

As per the outcome of this [governance poll](https://vote.makerdao.com/polling/QmZzFPFs#vote-breakdown), [Oasis.app](https://oasis.app/) will be whitelisted on the following OSM Oracle if this executive proposal passes:
* rETH/USD

Please review [MIP10](https://mips.makerdao.com/mips/details/MIP10) and the [forum thread](https://forum.makerdao.com/t/mip10c9-sp31-proposal-to-whitelist-oasis-app-on-rethusd-oracle/18195) to inform your position on this issue before voting.

### BlockTower Legal and Risk Work SPF

> As per this [Governance Poll](https://vote.makerdao.com/polling/QmcV2pMZ), **258,000 DAI** will be transferred to $multi-sig_address to support onboarding BlockTower Credit as a RWA Arranger, if this executive proposal passes.

Reviewer: now in sheet - 0x117786ad59BC2f13cf25B2359eAa521acB0aDCD9

> As per this [Governance Poll](https://vote.makerdao.com/polling/QmcV2pMZ), **258,000 DAI** will be transferred to $multi-sig_address to support onboarding BlockTower Credit as a RWA Arranger, if this executive proposal passes.

Reviewer: ```suggestion
As per this [Governance Poll](https://vote.makerdao.com/polling/QmcV2pMZ), **258,000 DAI** will be transferred to $multi-sig_address to support legal and risk work related to the onboarding of BlockTower Credit as a RWA Arranger, if this executive proposal passes.
```
